glusterd2/volume/store-utils.go

```diff
@@ -189,14 +189,17 @@ func GetAllBricksInCluster() ([]brick.Brickinfo, error) {
 }
 
 // AreReplicateVolumesRunning retrieves the volinfo objects from GetVolumes() function
-// and checks if all replicate, Disperse volumes are stopped before
-// stopping the self heal daemon.
-func AreReplicateVolumesRunning() (bool, error) {
+// and checks if all replicate, Disperse volumes are stopped except input volume ID
+//before stopping the self heal daemon.
+func AreReplicateVolumesRunning(ID uuid.UUID) (bool, error) {
```

Comment:
This variable can be named better - `skipPeer` or `ignorePeer`

Can you describe why the peer is skipped ? I'd like to understand this better

Reply:
This is volume ID, not the peerID.  to check any replicate volumes are running we get volumes from the store, as this volume is not yet updated in the store, need to skip this volume.